main.rb

```diff
@@ -0,0 +1,8 @@
+require_relative './app'
```

Comment:
- [ ] You did great job. However, If I run the app I'm getting several errors. Kindly fix them to make your project approve. In order to check your project, your app need to be fully functional and not showing errors. You can see errors in this video:

https://user-images.githubusercontent.com/69011963/198720498-e78895e9-bf7f-4fe6-8aa5-76f46b231a32.mov